samples/VisualBasic/SourceGenerators/GeneratedDemo/bin/Debug/netcoreapp3.1/GeneratedDemo.deps.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
❗ This pull request must be rebased to remove the bin/ and obj/ directories from _all commits_ involved in this pull request (we cannot have them in the repository history).

Reply:
Didn't realize there wasn't a git.ignore for this project, so for that I apologize for assuming... I can redo... however... given the other comment... doesn't sound like there is interest in accepting the pull.

Reply:
feel free to add obj/bin folders to the current .gitignore file.